CenterForOpenScience/osf.io
# [PLAT-761] markdown-it-mfr

## Purpose

We want to spin-off the wiki markdown-it-video `@[osf](....)` tag into it's own plugin with more flexible regex for the staging server etc.

Also there's a usability issue where the wiki markdown become particularly aggravating because pressing backspace sends you back a page in Firefox. 

## Changes

- pass internal domain and mfr domain to js contexVars
- disable backspace sending you  back a page
- adds markdown-it-mfr and downgrades markdown-it-video to a version with no `@[osf](....)` tag. 
- adds to yarn.lock

## QA Notes

This should allow `@[osf](....)` tags on staging to only staging and `@[osf](....)` tags prod to only access prod.

Also you should be able to click outside of the textbox and press backspace and not get sent back a page.

## Side Effects

None that I know of.

## Ticket

https://openscience.atlassian.net/browse/PLAT-761
